docs/release-1.2/existing-rhdh.md

```diff
@@ -187,37 +246,8 @@ For the `*-app-config` ConfigMap add the database configuration if isn't already
           host: ${POSTGRES_HOST}
           port: ${POSTGRES_PORT}
 ```
-If persistence is enabled (which should be the default setting), ensure that the PostgreSQL environment variables are accessible.
-The RHDH instance will be restarted automatically on ConfigMap changes.
-
-Optionally, include the plugin-notifications-backend-module-email-dynamic to fan-out notifications as emails.
-The environment variables below need to be provided to the RHDH instance.
-See more configuration options for the plugin [here](https://github.com/backstage/backstage/blob/master/plugins/notifications-backend-module-email/config.d.ts).
-
-```
-- disabled: false # 
-    package: "@redhat/plugin-notifications-backend-module-email-dynamic@1.2.0"
-    integrity: sha512-dtmliahV5+xtqvwdxP2jvyzd5oXTbv6lvS3c9nR8suqxTullxxj0GFg1uU2SQ2uKBQWhOz8YhSmrRwxxLa9Zqg==
-    pluginConfig:
-      notifications:
-         processors:
-           email:
-             transportConfig: # these values needs to be updated.
-               transport: smtp
-               hostname: ${NOTIFICATIONS_EMAIL_HOSTNAME}
-               port: 587
-               secure: false
-               username: ${NOTIFICATIONS_EMAIL_USERNAME}
-               password: ${NOTIFICATIONS_EMAIL_PASSWORD}
-             sender: sender@mycompany.com
-             replyTo: no-reply@mycompany.com
-             broadcastConfig:
-               receiver: "none"
-             concurrencyLimit: 10
-             cache:
-               ttl:
-                 days: 1
-```
+> Note: `${BACKEND_SECRET}` and `${RHDH_ROUTE}` variables are not by default accessible by backstage, so the values should be used directly in the ConfigMap or made accessible to backstage. 
+The `${POSTGRES_*}` variables *are* accessible by default, so they can be left in variable form.
 
 ### Import Orchestrator's software templates
 To import the Orchestrator software templates into the catalog via the Backstage UI, follow the instructions outlined in this [document](https://backstage.io/docs/features/software-templates/adding-templates).
```

Comment:
Let's mention in this context that the templates relies on Github as the git repository system, Quay as the image registry and the GitOps tools are ArgoCD and Tekton, so it will be clear to the user if the template will fit theirs infrastructure or they will have to change it for fit their tools.